bugbug/github.py

```diff
@@ -32,6 +32,10 @@ def get_issues() -> Iterator[IssueDict]:
     yield from db.read(GITHUB_ISSUES_DB)
 
 
+def delete_issues(match):
```

Comment:
Can you add type annotations here? It should be something like `def delete_issues(match: Callable[IssueDict, bool]) -> None`.

(you'd need to define IssueDict like https://github.com/mozilla/bugbug/blob/e9106d8b2e1889b49fd0177d8f4537c83383d853/bugbug/bugzilla.py#L19).

Reply:
Oh right, thanks 👍